### PR TITLE
test: Updated unit tests and minor fixes

### PR DESCRIPTION
**Pull request by abates**

### CODE REVIEW

File: nautobot_design_builder/tests/test_changes.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        current = {"key1": "initial-value", "key2": "new-value", "key3": "key3-value"}
+        got = revert_changed_dict(current, original, changed)
+        self.assertDictEqual({"key1": "initial-value", "key3": "key3-value"}, got)
+
+    def test_missing_changed_value_without_current(self):
+        original = {"key1": "initial-value"}
+        changed = {"key2": "new-value"}
+        current = {"key2": "new-value", "key3": "key3-value"}
+        got = revert_changed_dict(current, original, changed)
+        self.assertDictEqual({"key1": "initial-value", "key3": "key3-value"}, got)
```
Comment (chadell) [inline nautobot_design_builder/tests/test_changes.py:30]: this raises an interesting point. 
I would like to have the option to only define a few keys of the dict, so the keys not impacted by the design (e.g. key1),I don't care about their value. Only the keys added by the design (e.g. key2) have to be moved back to previous state, either removed or changed.
But, on the other side, I would like to have a way to define the whole dict as a change, not only targeting a few of the keys.
Both options look like two different operations... how do you see this?

Comment (chadell) [reply nautobot_design_builder/tests/test_changes.py:30]: @abates , I saw you merged it without resolving this comment. does it mean I shall raise it as an issue?

Comment (abates) [reply nautobot_design_builder/tests/test_changes.py:30]: This change didn't add any new functionality, it only tested existing functionality. I honestly don't even understand what you are asking. If you think we should add a new feature then a feature request should be put in.

Comment (chadell) [reply nautobot_design_builder/tests/test_changes.py:30]: You are right, I got confused with the behavior

**Pull request merged**